zingolib/src/lightclient/send.rs

```diff
@@ -81,38 +111,133 @@ pub mod send_with_proposal {
 
     impl LightClient {
         /// Calculates, signs and broadcasts transactions from a proposal.
```

Comment:
comment seems out of date and is same as following fn

Reply:
updated